.github/workflows/create_shared_dependencies_release_tag.yaml

```diff
@@ -0,0 +1,33 @@
+name: Create additional tag for each release with google-cloud-shared-dependencies version
```

Comment:
Would you make it generic so that gax and api-common-java versions are also tagged?

```
for ARTIFACT_ID in gax google-cloud-shared-dependencies api-common; do
...
```

(Approving, just in case you don't have time for it.)

Reply:
good idea - updated. thanks!